src/calculateRentalCost.js

```diff
@@ -29,6 +29,28 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const costDay = 40;
+  const weekDiscount = 50;
+  const threeDayDiscount = 20;
+  const week = 7;
+  const threeDay = 3;
+  let total = 0;
```

Comment:
```suggestion
  const total = costDay * days;
```
![image](https://github.com/mate-academy/js_task-transportation-on-vacation/assets/96409563/173ca8c1-46fc-43ce-a9b4-e55a741da885)